docs/how_to/data_intake/files.md

```diff
@@ -0,0 +1,92 @@
+# How to access files
+
+:::{admonition} What does this guide solve?
+:class: important
+This guide shows you how to use a local or remote file as a source for your dashboard.
+:::
+
+## Overview
+Lumen can read from multiple [data source types](../../reference/source).
+The [`FileSource`](../../reference/source/FileSource) type supports reading from either local or remote files of various formats: CSV, XLSX, XLS, Parquet, and JSON.
+
+## Local files
+One of the easiest ways to get a source is by using local files, such as a CSV file.
+Below is an example of how to add a local file as a source.
+
+::::{tab-set}
+:::{tab-item} YAML
+:sync: yaml
+``` {code-block} yaml
+:emphasize-lines: 1-5
+sources:
+  local_source:
+    type: file
+    tables:
+      local_table: local_table.csv
+
+targets:
+  - title: Table
+    source: local_source
+    views:
+      - type: table
+        table: local_table
+```
+:::
+
+:::{tab-item} Python
+:sync: python
+``` {code-block} python
+from lumen.pipeline import Pipeline
+
+data_path = "local_table.csv"
+pipeline = Pipeline.from_spec(
+    {
+        "source": {"type": "file", "tables": {"local_table": data_path}},
+    }
+)
+pipeline.data  # preview the data in a notebook
```

Comment:
I'd probably prefer to demonstrating the `Source` API directly instead of introducing `Pipeline` here. So let's just instantiate a Source and show off `source.get(table)`. Same thing below.

Reply:
I don't see any advantages in using the Source API over something like pandas. I think the power of lumen is the continual link from data to a dashboard, which pipelines provide. 

Yes this example is pretty basic and only uses a fraction of lumens functionality, but if `pipelines` is used throughout the documentation, users can combine these small examples into a larger complex Lumen Dashboard. 

Though it could be, I'm just not seeing the bigger picture here, and something similar could be done with the Source API.

Reply:
My basic problem is that the number of concepts covered in a particular how to guide should be minimal. Pipelines certainly are more useful than a source on its own but in this example it doesn't add anything and introduces additional complexity. The guide is meant to answer "how to access file data in Lumen"? The answer shouldn't be you define a source and then wrap it in a pipeline.

Reply:
Then I'm more inclined to remove the python part of this how-to. Do you have any objections to that?